www/scripts/codecheckerviewer/util.js

```diff
@@ -14,6 +14,18 @@ function (locale, dom, style, json) {
     'July', 'August', 'September', 'October', 'November', 'December'
   ];
 
+  var tooltips = {
+    detectionStatus : "Detection statuses are calculated based on the previous "
+                    + "detection status values when the reports are stored "
+                    + "again with the same run name. When storing the results "
+                    + "of a run from scratch then each report will have "
+                    + "detection status of 'New'.",
+    numOfUnresolved : "Number of unresolved reports are calculated on the "
```

Comment:
I think it is easier to understand how the report count is calculated if we write it down like this:
number of non unique reports excluding resolved, false positive and intentional reports